<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletForm</class>
 <widget class="QDialog" name="WalletForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameMain">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <widget class="QFrame" name="frameContent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>421</width>
        <height>371</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxWallets">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>钱包列表</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonOK">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="ButtonEditWallet">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>170</y>
        <width>96</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>96</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>96</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>编辑钱包</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonNewWallet">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>96</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>96</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>96</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>新建钱包</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonImportWallet">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>96</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>96</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>96</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>导入钱包</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonDeleteWallet">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>96</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>96</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>96</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>删除钱包</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
